hintman.cabal

```diff
@@ -84,10 +84,10 @@ library
 --                         Hintman.Suggestion.TrailingNewline
 --                         Hintman.Suggestion.TrailingSpaces
```

Comment:
Sorry, but why everything is commented out?

Would that mean that some of the previous PR could be completely wrong or never build?